doc/ChangeLog.txt

```diff
@@ -31,11 +31,14 @@ is now supported by the the read_saif command.
   read_saif [-scope scope] filename   
 
 The report_checks -group_count option has been renamed to -group_path_count.
-The report_checks -endpoing_count option has been renamed to -endpoint_path_count.
+The report_checks -endpoint_count option has been renamed to -endpoint_path_count.
 
-The report_checks -field hierarchical_pins field reports hierarical pins between
+The report_checks -field hierarchical_pins field reports hierarchical pins between
 a driver and a load in the path report.
 
+The suppress_msg and unsuppress_msg commands allow suppression/unsuppression of
+warnings/errors by code.
```

Comment:
explain where to find the code since it is not printed

Reply:
"Message codes can be found in doc/messages.txt." Added similar language to the docs.